packages/presets/src/blocks/_common/utils.ts

```diff
@@ -26,3 +26,30 @@ export async function markDownToDoc(host: EditorHost, answer: string) {
   }
   return doc as Doc;
 }
+
+export function formatDate(dateStr: string): string {
```

Comment:
maybe use i18nTime from '@affine/i18n';

Reply:
Get it. I will update it when move these AI Chat Block related components to AFFiNE.